staging/src/k8s.io/cloud-provider/app/controllermanager.go

```diff
@@ -85,6 +89,11 @@ the cloud specific control loops shipped with Kubernetes.`,
 			verflag.PrintAndExitIfRequested()
 			cliflag.PrintFlags(cmd.Flags())
 
+			if err := logsapi.ValidateAndApply(logOptions, nil); err != nil {
```

Comment:
What's the reason for nil feature gate here?

Reply:
[docs](https://pkg.go.dev/k8s.io/component-base@v0.26.1/logs/api/v1#ValidateAndApply) say "If nil, the default for these features is used", which sounds like a good fit here?

Reply:
e.g. metrics-server does the same: https://github.com/kubernetes-sigs/metrics-server/blob/master/cmd/metrics-server/app/options/options.go#L59

Reply:
metrics-server doesn't inherit and default feature gates from Kubernetes though -- I think at the very least we want to use the default feature gate we register here: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/cloud-provider/app/controllermanager.go#L66

Reply:
`utilfeature.DefaultFeatureGate` is actually better because it doesn't have to be mutable.

Reply:
changed and did a short `go test ./...` below `staging/src/k8s.io/cloud-provider`, but did not yet test any more

Reply:
Beware that if a non-nil feature gate instance is passed, then the component-base/logs features must have been added to it.

Reply:
@pohly added it to the DefaultMutableFeatureGate, same as `component-base/metrics/features` is added already

Reply:
Before I just rebase this PR again, is the way I implemented the feature gate ok?

Reply:
Yes, looks good to me.

Reply:
thanks :)

so another version prepared, freshly rebased and ready for your eyes @andrewsykim :)

Reply:
another ping @andrewsykim